src/Abc.Zebus.MessageDsl/Generator/SymbolNode.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Abc.Zebus.MessageDsl.Ast;
+
+namespace Abc.Zebus.MessageDsl.Generator;
+
+public class SymbolNode
```

Comment:
This is an implementation detail, so it should either be `internal` or (preferably) a nested `private` in `ProtoGenerator`, otherwise it would end up in the library's surface API.